whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/export/ExportFileWriterFactory.java

```diff
@@ -27,43 +26,33 @@ class ExportFileWriterFactory {
     private static final String CURRENTSERIAL_NONAUTH_FILENAME = "RIPE-NONAUTH.CURRENTSERIAL";
 
     private final DummifierNrtm dummifierNrtm;
-    private final DummifierCurrent dummifierCurrent;
-
-    private final String legacyExternalExportDir;
+    private final String externalExportDir;
     private final String source;
     private final String nonAuthSource;
-    private final String externalExportDir;
     private final String internalExportDir;
 
     @Autowired
-    ExportFileWriterFactory(final DummifierNrtm dummifierNrtm, final DummifierCurrent dummifierCurrent,
+    ExportFileWriterFactory(final DummifierNrtm dummifierNrtm,
                             @Value("${dir.rpsl.export.internal}") final String internalExportDir,
                             @Value("${dir.rpsl.export.external}") final String externalExportDir,
-                            @Value("${dir.rpsl.export.external.legacy}") final String legacyExternalExportDir,
                             @Value("${whois.source}") final String source,
```

Comment:
If you are renaming the property, classname should also be renamed ? DummifyLegacy to DummifyCurrent or something? I would prefer DummifySplitFiles if this dummifier is only used by splitfiles?

Reply:
Done